PennMobile/Notifications/NotificationPreference.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NotificationPreference.swift
+//  PennMobile
+//
+//  Created by Dominic Holmes on 12/27/19.
+//  Copyright © 2019 PennLabs. All rights reserved.
+//
+
+import Foundation
+
+typealias NotificationPreferences = Dictionary<NotificationOption, Bool>
+
+enum NotificationOption: String, Codable, CaseIterable {
```

Comment:
Are you using `CaseIterable` anywhere? It doesn't seem so. If not, we should delete it.